### ISSUE TITLE
Fatal: UdpSockets.bind got 5 arguments, expected 4

### ISSUE THREAD

**Issue by jsdario** [tradle/react-native-udp#59]
* Android (and iOS with different versions between native and JS)
* react-native 0.47

How? Rerun `npm install` and apparently my code jumped to the latest version.

Maybe this commit https://github.com/tradle/react-native-udp/commit/822b1e09337fded4d63baab0db3e0a86724b46eb is passed through the native-js bridge with extra params, crashing on Android.

**Comment by mvayngrib** [comment]
@jsdario oops, good catch, will release patch in a bit

**Comment by mvayngrib** [comment]
fixed in 2.2.1

@jsdario it's a good thing for this library that 2.2.0 broke your code, but to prevent this kind of "automatica" update from breaking ur project in the future, set your semver in package.json to allow patches but not minor versions to sneak through:

package.json
```
"some-library": "~x.x.x"  <-- allows patch upgrades
"some-library": "^x.x.x"  <-- allows minor version upgrades (dangerous)
```
